standalone-e2e/standalone-binary.test.ts

```diff
@@ -6,21 +6,25 @@ import {version} from '../package.json'
 const execPromise = promisify(exec)
 
 const isWin = process.platform === 'win32'
-
 const os = isWin ? 'win' : process.platform === 'darwin' ? 'darwin' : 'linux'
 
 const STANDALONE_BINARY = `datadog-ci_${os}-x64`
-
 const STANDALONE_BINARY_PATH = `${isWin ? '.\\' : './'}${STANDALONE_BINARY}${isWin ? '.exe' : ''}`
 
 const sanitizeOutput = (output: string) => output.replace(/(\r\n|\n|\r)/gm, '')
 
+const timeoutPerPlatform: Record<typeof os, number> = {
+  // Some macOS agents sometimes run slower, making this test suite flaky on macOS only.
+  // The issue is tracked here: https://github.com/actions/runner-images/issues/3885
+  darwin: 10 * 1000,
+  // Keep the default timeout for Linux.
+  linux: 5 * 1000,
+  // Running the binary on Windows is also slower than on Linux, and sometimes times out by a very small margin.
+  win: 6 * 1000,
```

Comment:
maybe at this point bumping this to 10 makes sense? The bottleneck is macos so it shouldn't be much of a difference

Reply:
(IIRC these tests run in parallel so macos is the bottleneck of the pipeline)

Reply:
True 😁